modal/image.py

```diff
@@ -285,7 +285,7 @@ def _from_args(
         build_function_input: Optional[api_pb2.FunctionInput] = None,
         image_registry_config: Optional[_ImageRegistryConfig] = None,
         context_mount: Optional[_Mount] = None,
-        force_build: bool = False,
+        force_build: Optional[bool] = False,
```

Comment:
Would it ever be None? (seeing as it's now an optional?)

Reply:
No, changed it!